plugin/pkg/admission/limitranger/admission.go

```diff
@@ -68,7 +86,21 @@ func (l *limitRanger) Admit(a admission.Attributes) (err error) {
 
 // NewLimitRanger returns an object that enforces limits based on the supplied limit function
 func NewLimitRanger(client client.Interface, limitFunc LimitFunc) admission.Interface {
-	return &limitRanger{client: client, limitFunc: limitFunc}
+	indexer := cache.NewIndexer(cache.MetaNamespaceKeyFunc, cache.Indexers{"namespace": cache.MetaNamespaceIndexFunc})
+	reflector := cache.NewReflector(
+		&cache.ListWatch{
```

Comment:
I'd be fine with that, though that is a bigger change which I'll wait until another PR for.